app/src/main/java/org/wikipedia/edit/EditSectionActivity.java

```diff
@@ -610,7 +616,7 @@ public void finishActionMode() {
     @Override
     protected void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
```

Comment:
Can we save the temporary wikitext here, instead of in `onPause`?
By saving it in `onPause`, it will get saved unnecessarily, e.g. when the user simply backs out of editing.

Reply:
Sure and done!